integration-tests/MySQLDockerfile

```diff
@@ -121,7 +121,7 @@ RUN service postgresql start
 
 # install mysql_fdw
 WORKDIR /mysql-client-tests/mysql_fdw
-RUN git clone https://github.com/EnterpriseDB/mysql_fdw --branch REL-2_7_0
+RUN git clone https://github.com/EnterpriseDB/mysql_fdw
```

Comment:
I think we should still pick a release, or else the tests are non-deterministic and non-reproducible. I can see the argument for wanting to know if tip breaks, but it's better to pin dependencies in CI at least.